### PR TITLE
Fetch procedure content from db

### LINKED ISSUE #43: Fetch procedure body
Be able to fetch and display the body of a procedure to show in the frontend

### PR DESCRIPTION
**Pull request by adityavidyadharan**
Closes #43 

Creates a function to fetch the content of a given procedure from the DB.

### CODE REVIEW

File: src/db/Procedures.ts [hunk 1]
```diff
@@ -36,13 +36,7 @@ export default class Procedures {
       user: 'kpmg',
       database,
     });
-    client.connect((err: any) => {
-      if (err) {
-        console.error('connection error', err.stack);
-      } else {
-        console.log('connected');
-      }
-    });
```
Comment (alex-hobby) [inline src/db/Procedures.ts:45]: Why do we not need these error messages?

File: src/db/Procedures.ts [hunk 2]
```diff
@@ -62,13 +56,7 @@ export default class Procedures {
...
```
Comment (alex-hobby) [inline src/db/Procedures.ts:71]: Same as above

**Review by alex-hobby — APPROVED** [review]
Looks good, see comments on error message handling

File: src/db/Procedures.ts (hunk 1 above)
Comment (adityavidyadharan) [reply src/db/Procedures.ts:45]: We should be doing this with a custom logger rather than just printing to console (creates linting error)

**Pull request merged**